README.md

```diff
@@ -142,6 +142,12 @@ You can generate an update site using [Maven](http://maven.apache.org/):
 This will create an update site in
 _com.github.eclipsecolortheme.updatesite/target/repository_.
 
+By default, Eclipse Color Theme will be built against the Eclipse Mars
+version of the platform. You can build against either Eclipse Luna or
+Kepler by setting the "platform-version" property, for example:
+
+    mvn clean verify -Dplatform-version=kepler
+
 License
```

Comment:
And now these days, it's all too clear why it's important to build/test against all supported versions of the platform.

This plugin is quite simply broken on Mars.  I've installed it, and my plugin manager shows that it is installed, but navigating to `Window > Preferences > General > Appearance` no longer reveals a `Color Theme` section as it once did on previous versions.  It's unusable.

I really wish this PR had been merged a year ago so we wouldn't be in this boat.